docs/admins.rst

```diff
@@ -250,6 +250,20 @@ For example, you might want to set up an HTTP proxy:
   - name: NO_PROXY
     value: localhost,127.0.0.1
 
+Limiting image size
+'''''''''''''''''''
+
+It is doable to limit the binary image size before pushing to registry. If a
+binary image's size exceeds configured image size, it will be refused to be
+pushed. A typical configuration looks like::
+
+  "image_size_limit": {
+    "binary_image": 10000
+  }
+
+The value is the size in bytes of uncompressed layers. When either
+``binary_image`` or ``image_size_limit`` is omitted, or setting ``0`` to
+``binary_image``, the check will be skipped.
```

Comment:
@tkdchen; Sorry, I don't know ho to do a block diff in GFMD (not sure it's even possible...). Suggestion below:

```
You can check the binary image's size before it is pushed to a
registry. If it exceeds the configured size, the push will fail.

A typical configuration looks like::

  "image_size_limit": {
    "binary_image": 10000
  }

The value is the size in bytes of the uncompressed layers. When
either ``binary_image`` or ``image_size_limit`` is omitted, or
if ``binary_image`` is set to ``0``, the check will be skipped.
```